src/Client.php

```diff
@@ -123,23 +123,39 @@ public function getOptions(): Options
     /**
      * {@inheritdoc}
      */
-    public function captureMessage(string $message, ?Severity $level = null, ?Scope $scope = null): ?EventId
+    public function captureMessage(string $message, ?Severity $level = null, ?Scope $scope = null/*, ?EventHint $hint = null*/): ?EventId
```

Comment:
The client is `final`, why are we doing this dance here? Isn't having a default value enough for BC?

Reply:
No, because the methods must have the same signature of the ones in the interface